Mathlib/CategoryTheory/Category/Cat/Adjunction.lean

```diff
@@ -51,4 +49,42 @@ def typeToCatObjectsAdj : typeToCat ⊣ Cat.objects where
           obtain rfl := Discrete.eq_of_hom f
           aesop_cat ) }
 
+
+/-- The connected components functor -/
+def connectedComponents.{v} : Cat.{v, u} ⥤ Type u where
```

Comment:
The universe `v` should be declared at the top of the file and should not appear in the name of this declaration.